### PR TITLE
Add in direct exchanges and routing keys for celery queues

### PR DESCRIPTION
**Pull request by elainearbaugh**

### CODE REVIEW

**Comment by elainearbaugh** [comment]
@mahendra confirmed on stage with the Exchanges/routing keys changes tasks are sent to the correct queues.

File: cabot/cabot_config.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 GRAPHITE_PASS = os.environ.get('GRAPHITE_PASS')
 GRAPHITE_FROM = os.getenv('GRAPHITE_FROM', '-10minute')
 JENKINS_API = os.environ.get('JENKINS_API')
+if not JENKINS_API.endswith('/'):
```
Comment (mahendra) [inline cabot/cabot_config.py]: Should we do it inside the code? In the Jenkins check ... (I am not a fan of placing code in config files - even though I am guilty of such crimes)

File: requirements.txt [hunk 2]
```diff
@@ -33,7 +33,7 @@ psycopg2==2.5.1
 pytz==2014.10
 redis==2.9.0
 requests==2.7.0
-six==1.5.1
+six==1.6.0
```
Comment (mahendra) [inline requirements.txt]: Why was this required?

File: cabot/cabot_config.py (hunk 1 above)
Comment (elainearbaugh) [inline cabot/cabot_config.py]: I put it here because 2 separate places in the code require a slash at the end:

```
./cabot/cabotapp/jenkins.py:    endpoint = settings.JENKINS_API + 'job/%s/api/json' % jobname
./cabot/cabotapp/templatetags/extra.py:    return '{}job/{}/'.format(settings.JENKINS_API, jobname
```
but if you think in the code would be better I can add a slash in both places

File: requirements.txt (hunk 2 above)
Comment (elainearbaugh) [inline requirements.txt]: setuptools v34.1.1 (used for installation of django-appconf) required six>=1.6.0...changed to pinning setuptools version

**Comment by elainearbaugh** [comment]
trigger build

File: cabot/cabot_config.py (hunk 1 above)
Comment (mahendra) [inline cabot/cabot_config.py]: I think we should fix it in code in both places. Ideally, we should never construct paths ourselves.

For eg: we should use os.path.join( ... ) instead of a + '/' + 'b' + ...
In this case we should use urlpath.urljoin

```
In [5]: urlparse.urljoin('http://booo', 'x/y')
Out[5]: 'http://booo/x/y'

In [6]: urlparse.urljoin('http://booo/', 'x/y')
Out[6]: 'http://booo/x/y'

In [7]: urlparse.urljoin('http://booo/', '/x/y')
Out[7]: 'http://booo/x/y'

In [8]: urlparse.urljoin('http://booo', '/x/y')
Out[8]: 'http://booo/x/y'
```

Comment (elainearbaugh) [inline cabot/cabot_config.py]: Much better, fixed!
